assets/dev/scss/frontend/widgets/playing-cards.scss

```diff
@@ -0,0 +1,211 @@
+.e-playing-cards-container {
+	--e-card-width: 220px;
+	--e-cards-flex-direction: row;
+	--e-cards-flex-justify-content: start;
+	--e-cards-flex-align-items: start;
+	--e-cards-transition-time: 300ms;
+	--e-cards-row-gap: 10px;
+	--e-cards-col-gap: 10px;
+	--e-card-border-radius: 10px;
+	--e-card-border: .7px solid #ccc;
+	--e-card-font-family: initial;
+	--e-card-font-size: 25px;
+	--e-card-font-weight: 300;
+	--e-card-diamond-color: red;
+	--e-card-heart-color: red;
+	--e-card-spade-color: black;
+	--e-card-club-color: black;
+	--e-card-suit-font-size: 35px;
+	--e-card-royal-suit-offset-h: 10%;
+	--e-card-royal-suit-offset-v: 25%;
+	--e-card-background-color: white;
+	--e-card-padding: 2px 4px;
+
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: var(--e-cards-flex-direction);
+	justify-content: var(--e-cards-flex-justify-content);
+	align-items: var(--e-cards-flex-align-items);
+	row-gap: var(--e-cards-row-gap);
+	column-gap: var(--e-cards-col-gap);
+	transition: var(--e-cards-transition-time);
+
+	.e-playing-cards-item * {
```

Comment:
You can nest this inside '.e-playing-cards-item'.

Reply:
Done